cellxgene_schema_cli/cellxgene_schema/ontology.py

```diff
@@ -1,86 +1,201 @@
 """Methods for working with ontologies and the OLS."""
-from urllib.parse import quote_plus
+import os
+import gzip
+import json
+
+
+class geneChecker:
```

Comment:
class names should be capitalized https://www.python.org/dev/peps/pep-0008/#class-names

Reply:
fixed in 4ac358decef7509c5944b70e93c9bd1e22eb45c1